src/aggregator/src/main/java/au/org/emii/util/DoubleRange.java

```diff
@@ -0,0 +1,13 @@
+package au.org.emii.util;
+
+public class DoubleRange {
+
+    public static Double start;
+    public static Double end;
```

Comment:
Did you mean public final?  static means these start and end fields will be shared between all instances of DoubleRange  e.g.

```
DoubleRange tenTo15 = new DoubleRange("10", "15");
DoubleRange twentyTo25 = new DoubleRange("20", "25");

System.out.println(tenTo15.start);
```

will print 20.0

final means they can never be changed after being set in the constructor and will be stored per instance if no static is used.